### PR TITLE
Tailwind JIT/purge fix

### PR DESCRIPTION
**Pull request by ryan-ludwig**
During development sometimes I'd add a new Tailwind class that hadn't been used yet and it wouldn't be added to the CSS file. Came across a few issues on Github with similar symptoms: https://github.com/tailwindlabs/tailwindcss/issues/4081. 
Then found some more info on the Tailwind JIT docs. https://tailwindcss.com/docs/just-in-time-mode#styles-don-t-update-when-saving-content-files

Bumping Tailwind and adding the `TAILWIND_MODE=watch` var seems to work for me now. Though not always with HMR.

### CODE REVIEW

File: tailwind.config.js [hunk 1]
```diff
@@ -3,13 +3,11 @@ const kvConfig = require('@kiva/kv-components/tailwind.config.js');
 module.exports = {
 	mode: 'jit',
 	presets: [kvConfig],
-	purge: {
```
Comment (mcstover) [inline tailwind.config.js:6]: This syntax previously "fixed" the purging... is the updated syntax you have specific to the new version?

File: package.json [hunk 2]
```diff
@@ -6,7 +6,7 @@
 	"private": true,
 	"license": "UNLICENSED",
 	"scripts": {
-		"dev": "cross-env NODE_ENV=development node server/dev-server.js",
+		"dev": "cross-env NODE_ENV=development TAILWIND_MODE=watch node server/dev-server.js",
```
Comment (mcstover) [inline package.json:9]: Happy to see this as I'd seen a similar "miss"

File: tailwind.config.js (hunk 1 above)
Comment (mcstover) [reply tailwind.config.js:6]: None of their examples are using this object with the `content` array now...

Comment (mcstover) [reply tailwind.config.js:6]: I think we got this syntax from the PurgeCss docs: https://purgecss.com/configuration.html#configuration-file

Comment (ryan-ludwig) [reply tailwind.config.js:6]: Yeah, I think it was actually the act of changing anything in the purge setting that caused Tailwind to pick up the new styles for the previous fix. Some people's workarounds to this bug were to add an empty directory to the list and save. Then remove it and save. To get Tailwind to regenerate all the classes.

**Review by mcstover — APPROVED** [review]
I'm cool with this if it's working well for you

File: tailwind.config.js (hunk 1 above)
Comment (eddieferrer) [reply tailwind.config.js:6]: > None of their examples are using this object with the `content` array now...

https://tailwindcss.com/docs/optimizing-for-production#enabling-manually

@ryan-ludwig @mcstover I think we still need the content array... I'll have to test My previous fix was specifically fixing the issue where you are using a class in UI that is not used in kv-ui-elements repo

**Comment by eddieferrer** [comment]
I did some additional tests:

Adding a new class does not seem to work on HMR for me. With `TAILWIND_MODE=watch` and `tailwind.config.js` from master it seems to work on both restart and HMR.

**Comment by ryan-ludwig** [comment]
@eddieferrer OK cool, thanks for testing! I'll try out reverting the changes in `tailwind.config.js` locally.

**Review by mcstover — APPROVED** [review]

**Comment by ryan-ludwig** [comment]
I still don't get the new classes built in HMR, but I wasn't getting that before either. So if it's working better for you with the original purge settings @eddieferrer let's roll with this.

**Review by emuvente — APPROVED** [review]

**Pull request merged**